subworkflows/local/call_repeat_expansions.nf

```diff
@@ -2,26 +2,78 @@
 // Run ExpansionHunter and Stranger
 //
 
-include { EXPANSIONHUNTER } from '../../modules/nf-core/expansionhunter/main'
-include { STRANGER } from '../../modules/nf-core/stranger/main'
+include { BCFTOOLS_NORM as SPLIT_MULTIALLELICS_EXP     } from '../../modules/nf-core/bcftools/norm/main'
+include { BCFTOOLS_REHEADER as BCFTOOLS_REHEADER_EXP   } from '../../modules/nf-core/bcftools/reheader/main'
+include { BCFTOOLS_VIEW as COMPRESS_STRANGER           } from '../../modules/nf-core/bcftools/view/main'
+include { EXPANSIONHUNTER                              } from '../../modules/nf-core/expansionhunter/main'
+include { PICARD_RENAMESAMPLEINVCF as RENAMESAMPLE_EXP } from '../../modules/nf-core/picard/renamesampleinvcf/main'
+include { STRANGER                                     } from '../../modules/nf-core/stranger/main'
+include { SVDB_MERGE as SVDB_MERGE_REPEATS             } from '../../modules/nf-core/svdb/merge/main'
+include { TABIX_BGZIPTABIX as BGZIPTABIX_EXP           } from '../../modules/nf-core/tabix/bgziptabix/main'
+include { TABIX_TABIX as INDEX_STRANGER                } from '../../modules/nf-core/tabix/tabix/main'
+include { TABIX_TABIX as TABIX_EXP_RENAME              } from '../../modules/nf-core/tabix/tabix/main'
 
 workflow CALL_REPEAT_EXPANSIONS {
     take:
         bam 			// channel: [ val(meta), path(bam), path(bai) ]
-        fasta 			// channel: /path/to/genome.fasta
         variant_catalog	// channel: /path/to/variant_catalog.json
+        case_info       // channel: [ case_id ]
+        fasta 			// channel: /path/to/genome.fasta
+        fai 			// channel: /path/to/genome.fasta.fai
 
     main:
         ch_versions = Channel.empty()
 
-        EXPANSIONHUNTER (bam, fasta, variant_catalog)
+        EXPANSIONHUNTER (
+            bam,
+            fasta,
+            variant_catalog
+        )
+
+        // Fix header and rename sample
+        BCFTOOLS_REHEADER_EXP (
+            EXPANSIONHUNTER.out.vcf.map{ meta, vcf -> [ meta, vcf, [] ]},
+            fai
+        )
+        RENAMESAMPLE_EXP ( BCFTOOLS_REHEADER_EXP.out.vcf )
+        TABIX_EXP_RENAME ( RENAMESAMPLE_EXP.out.vcf )
+
+        // Split multi allelelic
+        SPLIT_MULTIALLELICS_EXP (
+            RENAMESAMPLE_EXP.out.vcf.join(TABIX_EXP_RENAME.out.tbi),
+            fasta
+        )
+        //TABIX_EXP_SPLIT ( SPLIT_MULTIALLELICS_EXP.out.vcf )
```

Comment:
I guess this comment/line can be removed? 😅

Reply:
oops missed that one